.github/pull_request_template.md

```diff
@@ -0,0 +1,36 @@
+# Issue Number
+
+You can say `closes #XX` to automatically close the relevant issue.
+If this is just partial work on the issue, just mention either `#XX` or `GH-XX` here, **without** the _closes_ keyword.
+
+# Type of Change
+
+Please delete options that are not relevant.
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] This change requires a documentation update
+
+# Details
+
+Describe the feature added / bug fixed / work done
+
+# Screenshots
+
+Of the feature working, or the bug no longer existing, or anything else that's relevant.
+
+# How Has This Been Tested?
+
+Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration
+
+# Checklist
+
```

Comment:
This looks good for now. I might add some npm scripts which run some bug spotting and style checking processes, as well as `npm build` and add that to the checklist as something to run before creating a PR, but for now this is good.